src/site/content/en/blog/a11y-tips-for-web-dev/index.md

```diff
@@ -0,0 +1,557 @@
+Can you understand all important information
+and interact with the component using the screen reader and keyboard alone?
+
+The following questions should help you address screen reader accessibility.
+
+### Do all components and images have meaningful text alternatives?
+
+Wherever information about the _name_ or _purpose_
+of an interactive component is conveyed visually,
+provide an accessible text alternative.
+
+For example, if your `<fancy-menu>` UI component only displays a gear icon
+to indicate that it's a settings menu,
+it needs an accessible text alternative, such as "settings,"
+that conveys the same information.
+Depending on context,
+you can provide a text alternative using an `alt` attribute,
+an `aria-label` attribute, an `aria-labelledby` attribute,
+or plain text in the Shadow DOM.
+You can find general technical tips in [WebAIM Quick Reference](http://webaim.org/resources/quickref/).
+
+Any UI component that displays an image should provide a mechanism
+for providing alternative text for that image, analogous to the `alt` attribute.
+
+### Do your components provide semantic information?
+
+Assistive technology conveys semantic information
+that is otherwise expressed to sighted users via visual cues
+such as formatting, cursor style, or position.
+Native elements have this semantic information built-in by the browser,
+but for custom components you need to use
+[ARIA](http://www.w3.org/WAI/PF/aria/) to add the information.
+
+As a rule of thumb, any component that listens to a mouse click or hover event
+should not only have some kind of keyboard event listener,
+but also an ARIA role and potentially ARIA states and attributes.
+
+For example, a custom `<fancy-slider>` UI component might take an ARIA role of slider,
+which has some related ARIA attributes: `aria-valuenow`, `aria-valuemin` and `aria-valuemax`.
+By binding these attributes to the relevant properties on your custom component,
+you can allow users of assistive technology to interact with the element,
+change its value, and even cause the element's visual presentation to change accordingly.
+
+<figure class="w-figure w-figure--center">
+  <img src="./slider.png" class="w-screenshot" alt="A screenshot of a slider.">
+  <figcaption class="w-figcaption">
+    A range slider component.
+  </figcaption>
+</figure>
+
+```html
+<fancy-slider role="slider" aria-valuemin="1" aria-valuemax="5" aria-valuenow="2.5"></fancy-slider>
+```
+
+### Can users understand everything without relying on color?
+
+Color shouldn't be used as the only means of conveying information,
+such as indicating a status, prompting the user for a response,
+or distinguishing a visual custom component.
+For example, if you create a `<fancy-map>` component that uses color
+to distinguish between heavy, moderate, and light traffic,
+an alternative means of distinguishing traffic levels should also be made available.
+One solution might be to hover over an element to display information in a tooltip.
```

Comment:
Hey, @addyosmani,
After working on a social studies edu product for a while, I feel like chloropleths are the boss battle of contrast and color blindness: the strategy you take depends on how complex your map is and involves strategic compromises. For example, if your map has three or fewer data categories, you can use a tool like [Color Brewer](http://colorbrewer2.org/) to choose a color palette that works for various visual impairments. For more colors than that, you have to start doing things like what you propose, though you have to make sure they work for all input modalities. 

To avoid getting into all that complexity, I might propose choosing a more straightforward example, but up to you!

Reply:
Replaced this example with the pie chart example from WCAG